src/ui/starred-files-tab.h

```diff
@@ -48,7 +49,7 @@ private slots:
     QWidget *loading_failed_view_;
     QWidget *empty_view_;
 
-    GetStarredFilesRequest *get_starred_files_req_;
+    QScopedPointer<GetStarredFilesRequest> get_starred_files_req_;
```

Comment:
I see `QScopedPointer` is used in multiple places to avoid memory leak. Can we make add a bool field to SeafileApiRequest, and make it delete itself after the request is finished, if that field is true?